tools/UnquotedListAssociations/2016-11-27-unquoted_list_associations_log.txt

```diff
@@ -0,0 +1,2967 @@
+..code\datums\outfit.dm 
+list(path = count)
+list(otherpath = count)
+Total Unquoted List Associations: 2
+..code\game\gamemodes\blob\theblob.dm 
+list(bullet = 0)
+list(laser = 0)
+list(energy = 0)
+list(bomb = 0)
+list(bio = 0)
+list(rad = 0)
+list(fire = 80)
+list(acid = 70)
+Total Unquoted List Associations: 8
+..code\game\gamemodes\blob\blobs\core.dm 
+list(bullet = 0)
+list(laser = 0)
+list(energy = 0)
+list(bomb = 0)
+list(bio = 0)
+list(rad = 0)
+list(fire = 75)
+list(acid = 90)
+Total Unquoted List Associations: 8
+..code\game\gamemodes\blob\blobs\node.dm 
+list(bullet = 0)
+list(laser = 0)
+list(energy = 0)
+list(bomb = 0)
+list(bio = 0)
+list(rad = 0)
+list(fire = 65)
+list(acid = 90)
+Total Unquoted List Associations: 8
+..code\game\gamemodes\blob\blobs\shield.dm 
+list(bullet = 0)
+list(laser = 0)
+list(energy = 0)
+list(bomb = 0)
+list(bio = 0)
+list(rad = 0)
+list(fire = 90)
+list(acid = 90)
+Total Unquoted List Associations: 8
+..code\game\gamemodes\changeling\powers\mutations.dm 
+list(bullet = 0)
+list(laser = 0)
+list(energy = 0)
+list(bomb = 0)
+list(bio = 0)
+list(rad = 0)
+list(fire = 90)
+list(acid = 90)
+list(bullet = 0)
+list(laser = 0)
+list(energy = 0)
+list(bomb = 0)
+list(bio = 0)
+list(rad = 0)
+list(fire = 90)
+list(acid = 90)
+list(bullet = 40)
+list(laser = 40)
+list(energy = 20)
+list(bomb = 10)
+list(bio = 4)
+list(rad = 0)
+list(fire = 90)
+list(acid = 90)
+list(bullet = 40)
+list(laser = 40)
+list(energy = 20)
+list(bomb = 10)
+list(bio = 4)
+list(rad = 0)
+list(fire = 90)
+list(acid = 90)
+Total Unquoted List Associations: 32
+..code\game\gamemodes\clock_cult\clock_items\clockwork_armor.dm 
+list(bullet = 70)
+list(energy = 0)
+list(bomb = 60)
+list(bio = 0)
+list(rad = 0)
+list(fire = 100)
+list(acid = 100)
+list(bullet = 70)
+list(energy = 0)
+list(bomb = 60)
+list(bio = 0)
+list(rad = 0)
+list(fire = 100)
+list(acid = 100)
+list(bullet = 70)
+list(energy = 0)
+list(bomb = 60)
+list(bio = 0)
+list(rad = 0)
+list(fire = 100)
+list(acid = 100)
+Total Unquoted List Associations: 21
+..code\game\gamemodes\clock_cult\clock_scriptures\scripture_revenant.dm 
+list(time = 88)
```

Comment:
this isn't correctly-found; the line in question is `animate(invoker, color = list(rgb(255, 255, 255), rgb(255, 255, 255), rgb(255, 255, 255), rgb(0,0,0)), time = 88)` \, putting the found line outside of the list.